app/assets/stylesheets/global.scss

```diff
@@ -16,6 +16,8 @@ a {
   &:hover, &:focus {
     color: $black;
     text-decoration: none;
+    // keeping existing internal_link as it is, with this we could achieve this desired view on site as well
```

Comment:
Are these comments needed? Can we remove them if not please?

Reply:
i guess why not, it was just an attempt to explain how it could have been done, i'll adjust this as well, thanks :)